spec/DefinedSchemas.spec.js

```diff
@@ -0,0 +1,637 @@
+const { DefinedSchemas } = require('../lib/SchemaMigrations/DefinedSchemas');
+const Config = require('../lib/Config');
+
+const cleanUpIndexes = schema => {
+  if (schema.indexes) {
+    delete schema.indexes._id_;
+    if (!Object.keys(schema.indexes).length) {
+      delete schema.indexes;
+    }
+  }
+};
+
+fdescribe('DefinedSchemas', () => {
+  let config;
+  beforeEach(async () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000000;
+
+    config = Config.get('test');
+    await config.database.adapter.deleteAllClasses();
+  });
+  afterAll(async () => {
+    await config.database.adapter.deleteAllClasses();
+  });
+
+  describe('Fields', () => {
+    it('should keep default fields if not provided', async () => {
+      const server = await reconfigureServer();
```

Comment:
Not sure whether we still call `reconfigureServer` that before every test? IIRC most of these have been removed by @dplewis to speed up the test suite some months ago and are called onlywhere reconfig is explicitly necessary.

Reply:
How would you get  the `server` config params that `reconfigureServer()` returns ?

Reply:
Good questions, are these default params not accessible globally?